commerce-frontend-js/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -1,2 +1,66 @@
 @import "../js/side_panel/side_panel";
-@import "../js/step_tracker/step_tracker";
\ No newline at end of file
+@import "../js/step_tracker/step_tracker";
+@import "../js/info_box/info_box";
+
+.row.vertically-divided {
```

Comment:
selectors should be sorted alphabetically, rules should be sorted alphabetically, and indentation should be with tabs to match the `main.scss` files we have in other modules